src/centralstore/incremental/NodeBlock.cpp

```diff
@@ -221,6 +223,11 @@ NodeBlock* NodeBlock::get(unsigned int blockAddress) {
         node_block_logger.error("Error while reading edge reference data from block " + std::to_string(blockAddress));
```

Comment:
Don't we have to say which graph ID and which partition ID we are referring to here?

Reply:
I wish we could, But in this class, we don't have graph & partition information, Graph ID and Partition ID is used to create the NodeDB database file so it's in the file name, But in this class, all we have is the file stream [`std::fstream*`](https://github.com/miyurud/jasminegraph/blob/740e27dd015904c4fd14483546dcedafc25686e9/src/centralstore/incremental/NodeBlock.cpp#L264), to the DB file , But it seems we can't get the file path from `fstream` or at least it's not straightforward

- [SO Answer 1](https://stackoverflow.com/questions/3022692/how-get-file-path-from-stdifstream-in-c)
- [SO Answer 2](https://stackoverflow.com/questions/10773391/getting-filename-or-path-from-fstream)

Reply:
Based on the discussion we had today, it seems that it is not possible to link the graph ID and partition ID from the execution context. Hence we will keep this error message as it is and proceed.